contracts/Profile.sol

```diff
@@ -3,307 +3,91 @@
 pragma solidity ^0.8.0;
 
 import { Hub } from "./Hub.sol";
+import { HashingProxy } from "./HashingProxy.sol";
+import { Identity } from "./Identity.sol";
 import { IdentityStorage } from "./storage/IdentityStorage.sol";
-import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import { ParametersStorage } from "./storage/ParametersStorage.sol";
 import { ProfileStorage } from "./storage/ProfileStorage.sol";
-import { ShardingTable } from "./ShardingTable.sol";
+import { ADMIN_KEY, OPERATIONAL_KEY } from "./constants/IdentityConstants.sol";
 
 contract Profile {
-    event ProfileCreated(
```

Comment:
Not sure what are best practices, but for me it makes more sense to emit events from logic contracts so we can easily updated them in future. 

If we move this events to storage contracts, and we want later to add some more info to events, we would need to emit and listen for two events instead of just expanding one event.

Reply:
> Not sure what are best practices, but for me it makes more sense to emit events from logic contracts so we can easily updated them in future.
> 
> If we move this events to storage contracts, and we want later to add some more info to events, we would need to emit and listen for two events instead of just expanding one event.

Idea was to add basic event to the Storage, since it probably won't changes. Any changes in those events would mean that we changed Storage itself. Does it make sense to you?